From ceaba7d06758fdd130046cb66c014aada924cc25 Mon Sep 17 00:00:00 2001
From: Thomas Bechtold <tbechtold@suse.com>
Date: Thu, 12 Sep 2019 13:23:30 +0200
Subject: [PATCH] Build man pages for the commands that are documented

Running "sphinx-build -b man doc/source doc/build/man" allows the
build of man pages which are useful when working from the command
line.

Change-Id: Ia9e2bb2cc9d68bea71a19ff8296bbf9bd3cd95a4
---
 doc/source/conf.py | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/doc/source/conf.py b/doc/source/conf.py
index 25379905..ffa1d137 100644
--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -248,8 +248,10 @@ latex_use_xindy = False
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
 man_pages = [
-#    ('index', 'designate', u'Designate Documentation',
-#     [u'Managed I.T.'], 1)
+    ('cli/designate-manage', 'designate-manage', 'OpenStack DNSaaS',
+     ['OpenStack'], 1),
+    ('cli/designate-status', 'designate-status', 'OpenStack DNSaaS',
+     ['OpenStack'], 1),
 ]
 
 # If true, show URL addresses after external links.
-- 
2.23.0

